backend/mora/service/orgunit.py

```diff
@@ -46,6 +45,42 @@
                             url_prefix='/service')
 
 
+def _read_local_settings(unitid=None):
+    """ Read a set of settings from the database. The values are pr default
+    arbitrary strings, but we do reserve the words 'True' and 'False' for the
+    logic values True and False.
+    :param query: The query
+    """
+    conn = psycopg2.connect(user=settings.USER_SETTINGS_DB_USER,
```

Comment:
Please consider using a context manager, and factor connection creation into a separate helper method.